examples/powercli/datastore-usage-email/handler/script.ps1

```diff
@@ -33,22 +33,34 @@ if( ("$alarmName" -match "$($VC_CONFIG.VC_ALARM_NAME)") -and ([bool]($VC_CONFIG.
     } elseif($alarmStatus -eq "red") {
         $subject = "☢️ $($VC_CONFIG.EMAIL_SUBJECT) ☢️ "
         $threshold = "error"
-    }
+    } elseif($alarmStatus -eq "green") {
+        $subject = "$($VC_CONFIG.EMAIL_SUBJECT)"
+        $threshold = "normal"
+	}
```

Comment:
nit: Align "}"

Reply:
I don't understand what's happening here - the bottom brace is clearly aligned in VS Code, but not in the commit.

![image](https://user-images.githubusercontent.com/29929717/76137194-90de2c00-5fff-11ea-887f-5c33e5be5bc4.png)